contracts/governance/BaseMultisig.sol

```diff
@@ -160,16 +160,17 @@ contract BaseMultisig is IBaseMultisig {
 
         // Do not proceed with operation execution if insufficient votes.
         if (voteCount < signers.threshold) {
+            emit MultisigVoted(topic, msg.sender);
```

Comment:
Maybe add the vote count and threshold so it's easier to monitor?

Reply:
will do